server/server.js

```diff
@@ -46,6 +51,76 @@ app.get('/songDescription/:songId', async(req, res) => {
   }
 });
 
+app.post('/songDescription', async(req, res) => {
+  // console.log(req.body);
+  try {
+    const description = await db.saveDescriptions(req.body);
+    res.status(200).json({
+      success: true,
+      msg: 'create one song description',
+      data: description
+    });
+  } catch (error) {
+    res.status(400).json({
+      success: false,
+      msg: `something went wrong`,
+      error: error
+    });
+  }
+});
+
+app.put('/songDescription/:songId', async(req, res) => {
+  try {
+    const description = await db.findDescriptionandUpdate(req.params.songId, req.body, {
+      new: true,
+    });
+
+    if (!description) {
+      return res.status(400).json({
+        success: false,
+        msg: `No description for songId: ${req.params.songId}`
+      });
+    }
+    res.status(200).send({
+      success: true,
+      data: description
+    });
+  } catch (error) {
+    console.error(error);
+    res.status(400).json({
```

Comment:
Admittedly I haven't worked with `try...catch` blocks, so out of curiosity, does this `catch` block handle errors only for function calls in the `try` block, or does it handle server errors too, like a syntax error in your `try`?